### PR TITLE
Add handling for uploading periodic analytics to kissmetrics

### PR DESCRIPTION
**Pull request by TylerSheffels**
@dannyroberts 

CC: @gcapalbo

### CODE REVIEW

File: corehq/apps/analytics/tasks.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
+import calendar
```
Comment (dimagimon) [inline corehq/apps/analytics/tasks.py]: F401 'calendar' imported but unused

**Comment by dannyroberts** [comment]
:+1: once you're ready

**Comment by gcapalbo** [comment]
looks good, is there anything else we're waiting on for this one, or is it good to merge?

**Comment by TylerSheffels** [comment]
Good to merge by me

**Pull request merged**